docs/ports.md

```diff
@@ -0,0 +1,13 @@
+OpenShift Reserved Ports
+========================
+
+OpenShift reserves all ports on hosts that are reserved by [upstream Kubernetes](https://github.com/kubernetes/kubernetes/blob/master/pkg/master/ports/ports.go) as well as the following host ports:
+
+| 2379,2380 | masters | [etcd](https://github.com/openshift/installer) | etcd API |
```

Comment:
nit: Add a space after the comma here to match your "Node Exporter" line below?

Also, do you mean to link the installer for "etcd"?  I'd have expected a link to https://github.com/etcd-io/etcd#etcd-tcp-ports

Reply:
Yeah, although the installer / mcd controls where that goes.  So I was linking to the relevant authority within our space.